net.sourceforge.qtpfsgui.LuminanceHDR.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should be actually kept in code repository so all distributions can benefit.

Reply:
This will be since the next application release.